sknn/tests/test_data.py

```diff
@@ -60,8 +61,8 @@ def test_SetLayerParamsList(self):
         nn.set_parameters([(weights, biases)])
         
         p = nn.get_parameters()
-        assert_true((p[0].weights == weights).all())
-        assert_true((p[0].biases == biases).all())
+        assert_true((p[0].weights == weights.astype(theano.config.floatX)).all())
```

Comment:
Can we test both codepaths, 32- and 64-bit? So maybe create weights with `dtype` 32bit as argument to `uniform? Then below do the opposite?